jbcli/jbcli/utils/format.py

```diff
@@ -29,3 +31,17 @@ def echo_success(message):
     :type message: str
     """
     return secho(message, fg='green', bold=True)
+
+
+def human_readable_timediff(dt):
+    """ Generate a human readable time difference """
```

Comment:
This could have a better comment too :-D

> Generate a human readable time difference using the two largest time intervals, like "1 hour, 30 minutes ago"

Reply:
Some tests for this would be good too.